tools/cppcheck/suppressions-list

```diff
@@ -2,4 +2,4 @@ wrongmathcall:tests/unit-libm/test-libm.inc.h
 variableScope:jerry-libm/*.c
 invalidPointerCast:jerry-libm/*.c
 arithOperationsOnVoidPointer:jerry-libc/*.c
-commaSeparatedReturn:jerry-ext/include/jerryscript-ext/arg.impl.h
+commaSeparatedReturn:*
```

Comment:
Could you explain this change? I'm not sure suppressing a warning in the whole code base is a good idea.